.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 dist
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+examples
```

Comment:
I think we should commit the `examples` project. Just make sure that the directory is in `.npmignore` instead so we don't end up publishing that unnecessarily.